lib/stats/stats-query.c

```diff
@@ -28,107 +28,62 @@
 
 #include <string.h>
 
-static const gchar *_counter_type[SC_TYPE_MAX+1] =
+typedef struct _NamedStatsCounterItem
```

Comment:
This is `IndexedCounterItem`, but well... this index is used for get the name and we also provide a method to resolve the name by the index, so I'm ok with this.